db_creation.sql

```diff
@@ -4,7 +4,7 @@ CREATE DATABASE localProductions OWNER POSTGRES ENCODING = 'UTF8';
 
 
 -- Connect to thelocals db to create data for its 'public' schema
-\c localProductions
+\c localproductions --con la P maiuscola ritorna errore evidentemente in riga 3 non è case sensitive mentre qui sì.
```

Comment:
quindi CREATE DATABASE localProductions va bene, qui invece va tutto lowercase?

Reply:
In sostanza "CREATE DATABASE LOCALPRODUCTIONS" e "CREATE DATABASE localproductions" sono equivalenti, nella fase di connessione però il case sensitive è attivo.
Ovviamente dobbiamo modificare la linea di connessione anche negli altri script sql.